packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -1096,8 +1097,9 @@ class Auth {
             },
             {
                 ...(callbackURI && {callbackURI: callbackURI}),
+                ...(usid && {usid}),
```

Comment:
sending usid as part of the `authorizePasswordless` API call in an attempt to make the merge basket work